### DISCUSSION TITLE
Issues flashing YAML to Display

### DISCUSSION THREAD

**Discussion by jacky-coke** [Blackymas/NSPanel_HA_Blueprint#961] [Q&A] [answered]

**Comment by edwardtfn** [comment, marked as answer]
Tx should go to Rx, so you should switch brown and white in one side. 😉

**Comment by doron1** [comment]
You want to swap rx and tx on one side (only). So tx will connect to rx and
rx connects to tx.

Cheers

**Comment by jacky-coke** [comment]
Thanks a lot!
